juniper-vpn.py

```diff
@@ -246,7 +246,18 @@ def action_key(self):
 
     def action_continue(self):
         # Yes, I want to terminate the existing connection
+
+        # Fix up the broken HTML
+        html = self.r.read()
+        html = html.replace('checkSelected()",', 'checkSelected()"')
```

Comment:
Yeah it's not the best change - I was desperately trying to reconnect to the VPN at the time :) Thanks for the improvements.

As far as I can tell if there's no connection then we don't get this form. I certainly haven't had any problem so far.